src/components/sidenav/sidenav-theme.scss

```diff
@@ -1,3 +1,5 @@
 md-sidenav.md-THEME_NAME-theme {
-  background-color: '{{background-color}}';
+  &, md-content {
+    background-color: '{{background-hue-1}}';    
```

Comment:
Not, sure about `hue-1`. 

![image](https://cloud.githubusercontent.com/assets/4987015/13462733/0c22bb2e-e089-11e5-82e8-b45f83927ea6.png)

Shouldn't it be the primary background color?

![image](https://cloud.githubusercontent.com/assets/4987015/13462758/34618a84-e089-11e5-8412-d38df4110c0a.png)

Please correct me, if I'm wrong.